### DISCUSSION TITLE
General git question

### DISCUSSION THREAD

**Discussion by getlost01** [community/community#42467] [Other Feature Feedback, Questions, & Ideas] [answered]
I was asked a question related to git, can anyone explain it briefly.
```
The question is -
When do you use "git rebase" instead of "git merge"?
```

**Comment by jfullstackdev** [comment, marked as answer]
good coverage here I think so

https://www.edureka.co/blog/git-rebase-vs-merge/

though I've never used that

I only use GitHub Desktop without Git and here is my personal commit workflow that I'm using even in my company 

https://github.com/jdevstatic/my-github-desktop-commit-workflow
